aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -672,7 +865,7 @@ Kestrel listens on `http://localhost:5000` and `https://localhost:5001` (if a de
 
 *Replace the default certificate from configuration*
 
-<xref:Microsoft.AspNetCore.WebHost.CreateDefaultBuilder*> calls `serverOptions.Configure(context.Configuration.GetSection("Kestrel"))` by default to load Kestrel configuration. A default HTTPS app settings configuration schema is available for Kestrel. Configure multiple endpoints, including the URLs and the certificates to use, either from a file on disk or from a certificate store.
+`CreateDefaultBuilder` calls `options.Configure(context.Configuration.GetSection("Kestrel"))` by default to load Kestrel configuration. A default HTTPS app settings configuration schema is available for Kestrel. Configure multiple endpoints, including the URLs and the certificates to use, either from a file on disk or from a certificate store.
```

Comment:
IIRC because this paragraph is being made generic to apply to all versions. IIRC, `serverOptions` language seemed a bit 3.0-ish. Let's drop `options` and make it say ......

> `CreateDefaultBuilder` calls `Configure(context.Configuration.GetSection("Kestrel"))` by default to load Kestrel configuration.

That's generic for `CreateDefaultBuilder` across hosts.